pkg/interfaces/contracts/pool-hooks/IMevHook.sol

```diff
@@ -186,4 +210,24 @@ interface IMevHook {
      * @param newPoolMevTaxThreshold The new threshold to be set in a pool
      */
     function setPoolMevTaxThreshold(address pool, uint256 newPoolMevTaxThreshold) external;
+
+    /**
+     * @notice Checks if sender is MEV tax exempt.
+     * @dev A MEV tax exempt sender pays only the static swap fee percentage, regardless of the paid priority fee.
+     * @param sender Sender to check if is MEV tax exempt
+     * @return mevTaxExempt true if sender is MEV tax exempt
+     */
+    function isMevTaxExempt(address sender) external view returns (bool mevTaxExempt);
+
+    /**
+     * @notice Registers a list of senders as MEV tax exempt senders.
+     * @param senders Addresses of senders to be registered as MEV tax exempt
+     */
+    function addMevTaxExemptSenders(address[] memory senders) external;
```

Comment:
Do we really need to add/remove more than one at a time? How many are there? Would be simpler to just say `addMevTaxExemptSender(address sender)`.  Unless there are going to be dozens of them?

Reply:
I'm not super sure, but this might be handy in the future if we deploy to new chains and we need to do the setup from scratch.

The added complexity is not that great, so I'm leaning towards keeping it like this given the flexibility it brings.